### PR TITLE
Fix/model functions for new table

### PR DESCRIPTION
**Pull request by JustinPeczenij**
# Description
Hooking up new database to frontend

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Change Status
- [ ] Complete, tested, ready to review and merge

# How Has This Been Tested?
- [x] Plugging in frontend code to backend

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: api/allIncidents/incidentsModel.js [hunk 1]
```diff
@@ -18,29 +18,34 @@ module.exports = {
  * Returns all incidents in the db sorted by newest incident first
  */
 async function getIncidents() {
-  return await db('incidents')
-    .whereNot({ incident_date: null })
+  const incidents = await db('final_test')
+    .where({ status: 'approved' })
     .orderBy('incident_date', 'desc');
```
Comment (francishtknguyen) [inline api/allIncidents/incidentsModel.js:23]: Note: will need to change model to pull from new DS db

File: api/util/validationSchemas.js [hunk 2]
```diff
@@ -20,20 +20,20 @@ exports.incidentQuery = yup.object({
...
-  force_rank: yup.string().required().trim().min(2).max(50),
+  tweet_id: yup.string().trim().max(30).nullable(),
+  user_name: yup.string().trim().max(100).nullable(),
+  description: yup.string().required().trim().max(10000),
+  city: yup.string().trim().max(50).nullable(),
+  state: yup.string().trim().max(50).nullable(),
+  lat: yup.number().max(50).nullable(),
+  long: yup.number().max(50).nullable(),
+  title: yup.string().trim().max(50),
+  force_rank: yup.string().required().trim().max(50),
```
Comment (francishtknguyen) [inline api/util/validationSchemas.js:31]: great work fixing this bug caused by validation

**Review by francishtknguyen — DISMISSED** [review]

**Review by francishtknguyen — DISMISSED** [review]

File: api/middleware/checkIncidentExists.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
 const Incidents = require('../allIncidents/incidentsModel');
 
 const checkIncidentExists = (req, res, next) => {
-  const id = req.params.incident_id;
+  const id = Number(req.params.incident_id);
```
Comment (abe-one) [inline api/middleware/checkIncidentExists.js:4]: Strengthening of the code 👍

**Review by abe-one — DISMISSED** [review]
Code looks great 👍

**Review by francishtknguyen — DISMISSED** [review]

**Review by francishtknguyen — DISMISSED** [review]

**Review by abe-one — DISMISSED** [review]

**Review by francishtknguyen — APPROVED** [review]

**Review by abe-one — APPROVED** [review]

**Review by ryanhammer — APPROVED** [review]
Everything looks good

**Pull request merged**